recipes/openblas/all/conanfile.py

```diff
@@ -86,6 +169,8 @@ def package_info(self):
                 self.cpp_info.system_libs.append("pthread")
             if self.options.build_lapack:
                 self.cpp_info.system_libs.append("gfortran")
+        if self._is_msvc and tools.Version(self.settings.compiler.version) >= "14":
+            self.cpp_info.system_libs.append("legacy_stdio_definitions")
         self.cpp_info.names["cmake_find_package"] = "OpenBLAS"
         self.cpp_info.names["cmake_find_package_multi"] = "OpenBLAS"
```

Comment:
Could also take advantage of this PR to fix cmake imported target (`OpenBLAS::openblas`) please?

Reply:
I was wrong, imported target is `OpenBLAS::OpenBLAS`, but it also defines a component. This PR https://github.com/conan-io/conan-center-index/pull/2993 takes care of these conan generators things.